desec.py

```diff
@@ -19,6 +19,7 @@
 from datetime import datetime
 from enum import IntEnum
 from hashlib import sha256, sha512
+from logging import DEBUG, INFO, Formatter, StreamHandler, getLogger
```

Comment:
I'd prefer to just `import logging` and reference everything by the full name.

Reply:
1fb2d9ece163a3d987b5818d0e523574a8c18c9a